holoviews/element/util.py

```diff
@@ -235,6 +235,23 @@ def circular_layout(nodes):
     return (x, y, nodes)
 
 
+def quadratic_bezier(start, end, c0=(0, 0), c1=(0, 0), steps=50):
```

Comment:
Ideally this should be handled by the plotting extension. The main advantage I see of us doing this is that it helps consistency between matplotlib/bokeh. That said, plotting libraries might have more performant/robust code (though this is using numpy so it shouldn't be too slow).

Reply:
That would be nice. I might actually prefer holoviews handling this if the available bezier definitions in matplotlib/bokeh don't match up.

Reply:
Afaik they do match.